tensorflow/workspace.bzl

```diff
@@ -345,11 +345,11 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
     )
 
     PROTOBUF_URLS = [
```

Comment:
Perhaps add a comment that this is version 3.6.0. The SHA1 value is inscrutable.